src/coprocessor/statistics/cmsketch.rs

```diff
@@ -29,25 +29,23 @@ impl CmSketch {
     }
 
     // `hash` hashes the data into two u64 using murmur hash.
-    fn hash(mut bytes: &[u8]) -> (u64, u64) {
-        let mut out: [u8; 16] = [0; 16];
-        murmur3_x64_128(&mut bytes, 0, &mut out);
-        (
-            LittleEndian::read_u64(&out[0..8]),
-            LittleEndian::read_u64(&out[8..16]),
-        )
+    fn hash(mut bytes: &[u8]) -> Result<(u64, u64)> {
+        let out =
+            murmur3::murmur3_x64_128(&mut bytes, 0).map_err(|err| Error::Other(err.to_string()))?;
```

Comment:
Unwrap is OK as there can't be any IO error using bytes.

Reply:
okay